Parse/src/main/java/com/parse/ParseQuery.java

```diff
@@ -1699,6 +1701,31 @@ public void getInBackground(final String objectId, final GetCallback<T> callback
    * {@code ParseQuery}.
    * <p/>
    * This only works on keys whose values are {@link ParseObject}s or lists of {@link ParseObject}s.
```

Comment:
I thought so also when I copied the comment from `containsAll`, but was not sure to change it.

Thanks to comment this 👍  Someone could write a good documentation.